### PR TITLE
Round 2 of small type improvements

### PR DESCRIPTION
**Pull request by jeremywiebe**
## Summary:

A few more type fixes.

Issue: "none"

## Test plan:

✅ `yarn; yarn tsc` 
✅ `yarn test`

### CODE REVIEW

File: packages/perseus/src/widgets/__testdata__/expression.testdata.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const expressionItem3Options: PerseusExpressionWidgetOptions = {
             value: "x+1",
         },
         {
-            considered: "incorrect",
+            considered: "wrong",
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/__testdata__/expression.testdata.ts:74]: I checked the published data. We definitely only use the set of `"correct" | "wrong" | "invalid"` as seen here: https://github.com/Khan/perseus/pull/582/files#diff-5737e5c75b244a64356894bb42c711539f1589cfbd9dcbbd4b22a3db8773d524R338

File: packages/perseus/src/widgets/__tests__/expression.test.ts [hunk 2]
```diff
@@ -47,7 +45,6 @@ describe("Expression Widget", function () {
         jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
             testDependencies,
         );
-        registerAllWidgetsForTesting();
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/__tests__/expression.test.ts:50]: We call this function in `renderQuestion` and all of the tests in this file use that helper, so this is redundant.

File: packages/perseus-editor/src/widgets/expression-editor.tsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ const {InfoTip, PropCheckBox, TexButtons} = components;
 const {getDependencies} = Dependencies;
 
 // An answer can be considered correct, wrong, or ungraded.
-const CONSIDERED = ["correct", "wrong", "ungraded"];
+const CONSIDERED = ["correct", "wrong", "ungraded"] as const;
```
Comment (nedredmond) [inline packages/perseus-editor/src/widgets/expression-editor.tsx:21]: Can we use this pattern to re-use this type in `perseus-types.ts`? Or the other way around-- define there and use here?

Names are just to demonstrate.
```
export const consideredStrings = ["correct", "wrong", "ungraded"] as const;
export type considered = typeof consideredStrings[number];
```

**Review by nedredmond — COMMENTED** [review]
Love this. We can go further!

File: packages/perseus-editor/src/widgets/expression-editor.tsx [hunk 4]
```diff
@@ -433,8 +433,8 @@ class ExpressionEditor extends React.Component<any, any> {
 }
 
 // Find the next element in arr after val, wrapping around to the first.
-const findNextIn = function (arr: Array<string>, val: any) {
-    let ix = _(arr).indexOf(val);
```
Comment (handeyeco) [inline packages/perseus-editor/src/widgets/expression-editor.tsx:437]: Nice 👍

File: packages/perseus/src/interactive2.ts [hunk 5]
```diff
@@ -7,19 +7,16 @@ import KhanColors from "./util/colors";
 const Interactive2 = {
     MovablePoint: MovablePoint,
     addMovablePoint: function (graphie: any, options: any): any {
-        // @ts-expect-error [FEI-5003] - TS2350 - Only a void function can be called with the 'new' keyword.
```
Comment (handeyeco) [inline packages/perseus/src/interactive2.ts:10]: Nice 👍

File: packages/perseus/src/renderer.tsx [hunk 6]
```diff
@@ -1770,7 +1770,7 @@ class Renderer extends React.Component<Props, State> {
             return widgetIsGraded && !widgetIsStatic;
         });
 
-        const widgetScores: Record<string, any> = {};
+        const widgetScores: Record<string, PerseusScore> = {};
```
Comment (handeyeco) [inline packages/perseus/src/renderer.tsx:1773]: Like this 🌈

File: packages/perseus/src/widgets/__tests__/expression.test.ts [hunk 7]
```diff
@@ -15,9 +15,7 @@ import {Expression} from "../expression";
 
 import {renderQuestion} from "./renderQuestion";
 
-type Item = any;
```
Comment (handeyeco) [inline packages/perseus/src/widgets/__tests__/expression.test.ts:18]: Heck yeah

**Review by handeyeco — APPROVED** [review]
Love it, thank you!

**Pull request merged**